kubectl-fdb/cmd/root.go

```diff
@@ -38,6 +38,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var skipCommand = false
```

Comment:
Can we get rid of this global variable and make use of `PreRunE` instead of `PreRun`. Based on the documentation the whole chain should stop executing if `PreRuneE` is used and an error is returned.

Reply:
We have to make use of `PersistentPreRun` and return an error, then we can remove this global variable, see: https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1881#pullrequestreview-1729917480.